context/app/static/js/components/organ/Assays/Assays.jsx

```diff
@@ -54,9 +56,14 @@ function Assays({ organTerms }) {
   );
 
   const { searchData } = useSearchData(query);
```

Comment:
I think the simplest approach would just be to lift this to the parent for both `Assays` and `Samples` and avoid using the `useEffect` altogether?